crates/db_views_moderator/src/structs.rs

```diff
@@ -26,52 +26,52 @@ use serde::{Deserialize, Serialize};
 #[derive(Debug, Serialize, Deserialize, Clone)]
 pub struct ModAddCommunityView {
   pub mod_add_community: ModAddCommunity,
-  pub moderator: PersonSafe,
+  pub moderator: Option<PersonSafe>,
```

Comment:
These look good, now just to tweak the boxed queries to left joins as opposed to inner joins, so the mod has a chance to not come back when using hide_mod_names.